src/Dumplie/Infrastructure/Doctrine/Dbal/Metadata/TypeRegistry.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Infrastructure\Doctrine\Dbal\Metadata;
+
+use Doctrine\DBAL\Schema\Table;
+use Dumplie\Application\Metadata\Schema\FieldDefinition;
+use Dumplie\Infrastructure\Doctrine\Dbal\Metadata\Field\TextMapping;
+
+final class TypeRegistry
+{
+    /**
+     * @var TypeMapping[]
+     */
+    private $mapping;
+
+    public function __construct(array $typeMappings = [])
+    {
+        $this->register(new TextMapping());
```

Comment:
I dont think we should register default types here. Maybe additional static constructor will be better place for that?